test/test-lex.cpp

```diff
@@ -113,6 +113,62 @@ TEST_F(test_lex, lex_block_comments) {
   }
 }
 
+TEST_F(test_lex, lex_unopened_block_comment) {
+  {
+    error_collector v;
+    padded_string input(u8"hello */"_sv);
+    lexer l(&input, &v);
+    l.skip();  // identifier
+    l.skip();  // star
+    l.skip();  // slash
```

Comment:
Must fix: These comments are wrong. This code lexes as `identifier`, `slash`, `end_of_file`. There is no `star` token.

Must fix: There shouldn't be a `star` token if error_unopened_block_comment is emitted. The `*/` should be treated as a comment and ignored.